### ISSUE TITLE
Incomplete sentence to be corrected

### ISSUE THREAD

**Issue by ebeauzec** [MicrosoftDocs/windowsserverdocs#5326]
Bullet # 3 of "Technology Introduction" seems to be an incomplete sentence. 
Currently: "Easily change the compute-to-storage ratio in your hyper-converged I"
(Perhaps) it should be changed to: "Easily change the compute-to-storage ratio in your hyper-converged Infrastructure" ??

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cebe1cda-c758-b30b-bb43-161e1ca13b77
* Version Independent ID: 7fbdf25e-0ca7-f60d-8c0c-f12e18adafee
* Content: [Cluster Sets](https://docs.microsoft.com/en-us/windows-server/storage/storage-spaces/cluster-sets)
* Content Source: [WindowsServerDocs/storage/storage-spaces/Cluster-Sets.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/storage/storage-spaces/Cluster-Sets.md)
* Product: **windows-server**
* Technology: **storage-spaces**
* GitHub Login: @JohnMarlin-MSFT
* Microsoft Alias: **johnmar**

**Comment by illfated** [comment]
Fair point, looks like that sentence has been like that since it was written Jun 16, 2018 (commit  https://github.com/MicrosoftDocs/windowsserverdocs/commit/ae68fdac6eaaf3bf00630f3c9156bd2051ffdfcc). 

Reading the preceding text, I thought maybe it should have been something like "cluster" or similar, but I can confirm that the remaining letter at the end of the sentence is a capital I, not a lowercase l, so it looks like your suggestion is closer to the original idea.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
